pkg/cassandra/config/config.go

```diff
@@ -143,8 +158,9 @@ func (c *Configuration) NewCluster(logger *zap.Logger) (*gocql.ClusterConfig, er
 
 	if c.Authenticator.Basic.Username != "" && c.Authenticator.Basic.Password != "" {
 		cluster.Authenticator = gocql.PasswordAuthenticator{
-			Username: c.Authenticator.Basic.Username,
-			Password: c.Authenticator.Basic.Password,
+			Username:              c.Authenticator.Basic.Username,
+			Password:              c.Authenticator.Basic.Password,
+			AllowedAuthenticators: defaultApprovedAuthenticators,
```

Comment:
this needs to come from user input, the way I understood the ticket.

Reply:
ok so I will `AllowedAuthenticators ` to Authenticator struct
```sh
type Authenticator struct {
	Basic BasicAuthenticator `yaml:"basic" mapstructure:",squash"`
	// TODO: add more auth types
}
```

Reply:
currently i think i  have allowed all the default auth available so every possible auth would work 

> this needs to come from user input, the way I understood the ticket.